src/services/types.ts

```diff
@@ -1771,6 +1769,7 @@ export const enum ScriptElementKind {
     linkText = "link text",
 }
 
+/** @internal */
 export const enum ScriptElementKindModifier {

```

Comment:
This is used https://github.com/search?q=%2F%5Cbts%5C.ScriptElementKindModifier%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code